samples/gpio-pci-idio-16.c

```diff
@@ -109,6 +108,11 @@ main(int argc, char *argv[])
         err(EXIT_FAILURE, "failed to initialize device emulation");
     }
 
+    ret = lm_setup_log(lm_ctx, verbose ? _log : NULL, LM_DBG);
```

Comment:
I think this should be:

lm_setup_log(lm_ctx, _log, verbose ? LM_DBG : LM_ERR);

?

Reply:
Ack